### ISSUE TITLE
Transparency texture not showing

### ISSUE THREAD

**Issue by MagicalPaladin** [armory3d/armory#107]
![armorybugtrans1](https://cloud.githubusercontent.com/assets/15276930/21769159/8dcd1526-d67d-11e6-9c8a-5f70bcd89f89.png)
Just planes, not particiles, with a cloud baked texture
![armorybugtrans2](https://cloud.githubusercontent.com/assets/15276930/21769186/bc20b2c0-d67d-11e6-8118-e03971307004.png)
Clean cache does nothing

**Comment by luboslenco** [comment]
Heyo,

Thanks for report. Please throw me a .blend file as usual.

**Comment by MagicalPaladin** [comment]
http://pasteall.org/blend/index.php?id=45014 here you are !

**Comment by luboslenco** [comment]
Some investigation:

- Using Build 6 - set Camera - Properties - Armory Render Path - Translucency to On  (there are still issues with auto-translucency, more fixes in upcoming build..)
- Cloud texture does not seem to have alpha channel. I connected color socket instead as a workaround.
- Optional: for clouds material, set Properties - Armory Props - Advanced - Override Cull Mode to None. This way quads will be visible from both sides.

With those steps I got it to work. Hopefully the alpha texture can improve it further:)
![cloud](https://cloud.githubusercontent.com/assets/3685704/21955076/589c9b4e-da63-11e6-861a-6fa753ad9f92.jpg)
